skimage/registration/tests/test_phase_cross_correlation.py

```diff
@@ -145,7 +143,7 @@ def test_wrong_input():
         ]
     ):
         with pytest.raises(ValueError):
-            phase_cross_correlation(template, image, return_error=True)
```

Comment:
Until 0.22 we told users to pass `return_error='always'` to this function to silence the warning. Starting with 0.22 this will immediately return an error without any explanation. 

I kind of feel like removing the parameter completely requires another deprecation? It get that this is totally annoying but that's how I would interpret our deprecation guide lines and the user experience. Or am I confusing myself right now?

Reply:
See https://github.com/scikit-image/scikit-image/pull/7174